activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/conf/activiti/services/connectors/CloudConnectorsAutoConfiguration.java

```diff
@@ -21,24 +21,44 @@
 import org.activiti.engine.impl.bpmn.parser.factory.DefaultActivityBehaviorFactory;
 import org.activiti.engine.impl.persistence.entity.integration.IntegrationContextManager;
 import org.activiti.runtime.api.connector.IntegrationContextBuilder;
-import org.activiti.services.connectors.IntegrationContextMessageBuilderFactory;
 import org.activiti.services.connectors.behavior.MQServiceTaskBehavior;
+import org.activiti.services.connectors.message.IntegrationContextMessageBuilderFactory;
 import org.conf.activiti.runtime.api.ConnectorsAutoConfiguration;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.AutoConfigureBefore;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.cloud.stream.binder.rabbit.properties.RabbitProducerProperties;
+import org.springframework.cloud.stream.binding.BinderAwareChannelResolver.NewDestinationBindingCallback;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.ApplicationEventPublisher;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.PropertySource;
+import org.springframework.expression.Expression;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
 
 @Configuration
 @AutoConfigureBefore(value = ConnectorsAutoConfiguration.class)
 @PropertySource("classpath:config/integration-result-stream.properties")
 @ComponentScan("org.activiti.core.common.spring.connector")
 public class CloudConnectorsAutoConfiguration {
 
+    @Value("${activiti.spring.cloud.stream.integrationRequestSender.routing-key-expression}")
+    private String routingKeyExpression;
+
+    /**
+     * Configures routing key expression for dynamic cloud connector destinations
+     */
+    @Bean
+    public NewDestinationBindingCallback<RabbitProducerProperties> dynamicConnectorDestinationsBindingCallback() {
```

Comment:
@erdemedeiros I don't think so, but I should make it conditional on Rabbit Binder just in case...

Reply:
@erdemedeiros I made dynamicConnectorDestinationsBindingCallback bean `@Conditional` on `RabbitBinderProperties.class` just in case someone will want to use Kafka Binder.